resolwe_bio/processes/plots/dexseq_plot.yml

```diff
@@ -0,0 +1,75 @@
+# =============
+# Plot - DEXSeq
+# =============
+---
+
+- slug: plot-dexeq
```

Comment:
What is a typical running time for this process? Since its to be used for visualisations, I think its class should be set to `scheduling_class: interactive` and `persistence: TEMP`